### DISCUSSION TITLE
Is it possible to unconceal?

### DISCUSSION THREAD

**Discussion by helmecke** [nvim-treesitter/nvim-treesitter#2889] [Q&A]
Hi, I want to remove some `conceal` from `markdown`.
https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/markdown/highlights.scm#L78-L79

Is this possible without overriding the hole `highlights.scm` via `set_query()`?

**Comment by mosheavni** [comment]
I also want to remove the conceal on json files without disabling the `conceallevel` altogether.
Is it possible?
